src/Tribe/Commerce/PayPal/Attendance_Totals.php

```diff
@@ -70,20 +70,20 @@ protected function should_count( Tribe__Tickets__Ticket_Object $ticket ) {
 	 * @since 4.7
 	 */
 	public function print_totals() {
-		$total_sold_label = esc_html_x( 'Total Tickets Issued:', 'attendee summary', 'event-tickets' );
-		$total_paid_label = esc_html_x( 'Complete:', 'attendee summary', 'event-tickets' );
+		$total_sold_label      = _x( 'Total Tickets Issued:', 'attendee summary', 'event-tickets' );
+		$total_complete_label  = _x( 'Complete:', 'attendee summary', 'event-tickets' );
+		$total_cancelled_label = _x( 'Cancelled:', 'attendee summary', 'event-tickets' );
 
 		$total_sold      = $this->get_total_sold();
-		$total_paid      = $this->get_total_complete();
+		$total_complete  = $this->get_total_complete();
 		$total_cancelled = $this->get_total_cancelled();
 
-		echo "
-			<ul>
-				<li> <strong>$total_sold_label&nbsp;</strong>$total_sold </li>
-				<li> $total_paid_label&nbsp;$total_paid </li>
-				<li> $total_cancelled_label $total_cancelled </li>
+		?><ul>
```

Comment:
Should we move this towards an `admin-view` file?

Reply:
I guess we could here, it's small but might as well stick to precedence